src/FluentHandler.php

```diff
@@ -101,6 +101,32 @@ protected function processFormat(array $record, string $tag): string
         return $tag;
     }
 
+    /**
+     * returns the context
+     * @return array
+     */
+    protected function getContext($context)
+    {
+        if ($this->contextHasException($context)) {
+            return $this->getContextExceptionTrace($context);
+        }
+        return $context;
+    }
+
+    protected function contextHasException($context)
+    {
+        return (
+                is_array($context)
+            && array_key_exists('exception', $context)
+            && $context['exception'] instanceof \Exception
+        );
+    }
+
+    protected function getContextExceptionTrace($context)
```

Comment:
typehint `array` should be added on `$context`.
Please then add return type `string` on the method.